src/app/util/attribute-storage.cpp

```diff
@@ -171,12 +171,6 @@ void emberAfEndpointConfigure(void)
     {
         // This is assuming that EMBER_AF_ENDPOINT_DISABLED is 0
         static_assert(EMBER_AF_ENDPOINT_DISABLED == 0, "We are creating enabled dynamic endpoints!");
-        memset(&emAfEndpoints[FIXED_ENDPOINT_COUNT], 0,
-               sizeof(EmberAfDefinedEndpoint) * (MAX_ENDPOINT_COUNT - FIXED_ENDPOINT_COUNT));
-        for (ep = FIXED_ENDPOINT_COUNT; ep < MAX_ENDPOINT_COUNT; ep++)
-        {
-            emAfEndpoints[ep].endpoint = kInvalidEndpointId;
-        }
```

Comment:
We may want to keep this loop, to be safe, in case someone shuts things down without removing some dynamic endpoints.

Also might be good to explicitly set the bitmask to `EMBER_AF_ENDPOINT_DISABLED` (and take out that static_assert and comment).

Reply:
Did one better by just calling the constructor again (which should set up sane defaults):

```
    if (MAX_ENDPOINT_COUNT > FIXED_ENDPOINT_COUNT)
    {
        //
        // Reset instances tracking dynamic endpoints to safe defaults.
        //
        for (ep = FIXED_ENDPOINT_COUNT; ep < MAX_ENDPOINT_COUNT; ep++) {
            emAfEndpoints[ep] = EmberAfDefinedEndpoint();
        }
    }
```